### ISSUE TITLE
yarn audit fails

### ISSUE THREAD

**Issue by PhilippMundhenk** [GENIVI/iot-event-analytics#93]
Tried following prerequisites and tldr, but had some issues (see #92). Instead decided to build manually, without VSCode plugin:
```bash
$ docker-compose -f docker-compose/docker-compose.platform.yml up
[...]
Step 13/26 : RUN yarn audit
 ---> Running in edd24f1f4dba
yarn audit v1.19.1
warning Resolution field "underscore@1.12.1" is incompatible with requested version "underscore@~1.10.2"
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ ws                                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=6.2.2 <7.0.0 || >=7.4.6                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ ws                                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ ws                                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1748                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ ws                                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=6.2.2 <7.0.0 || >=7.4.6                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ mqtt                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ mqtt > ws                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1748                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ moderate      │ Regular expression denial of service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ glob-parent                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=5.1.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ mqtt                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ mqtt > help-me > glob-stream > glob-parent                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1751                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
3 vulnerabilities found - Packages audited: 420
Severity: 3 Moderate
Done in 1.35s.
The command '/bin/sh -c yarn audit' returned a non-zero code: 4
ERROR: Service 'configmanager' failed to build : Build failed
```

**Comment by llakie** [comment]
Will be fixed in  PR #97 which decreases the level yarn audit recognizes a CVE as error.

**Comment by llakie** [comment]
Merged. `yarn audit` will now only trigger on critical vulnerabilities
